.github/actions/docker-build/action.yaml

```diff
@@ -36,6 +36,15 @@ inputs:
   push:
     description: Whether to push the image
     required: true
+  platform:
+    description: "Target given CPU platform architecture (default: linux/amd64)"
```

Comment:
How do I list more than one? Or can this action only run on a single arch, but is then called by another workflow?

Reply:
more info at https://github.com/redhat-developer/rhdh/pull/2036/files/7b9d684bcf52bcf4ad16565736616256989ce678#r1914593017